packages/react-experiments/src/components/SelectedItemsList/SelectedItemsList.types.ts

```diff
@@ -145,4 +145,14 @@ export interface ISelectedItemsListProps<T> extends React.ClassAttributes<any> {
    * Should be used if it's possible to change some properties on items so a strict compare will fail
    */
   itemsAreEqual?: (item1?: any, item2?: any) => boolean;
+
+  /**
+   * A localized string to read out when a selection is deleted
+   */
+  selectionDeletedText?: string;
+
+  /**
+   * Callback when recipient(s) are deleted, to notify screen readers
+   */
+  onAnnounceDeletedRecipients?: (text?: string) => void;
```

Comment:
We shouldn't need this. There's already a callback for when items are deleted, consumers can use this to do whatever screenreader behavior they like

Reply:
Removed, and implemented the suggestion.